windows_container_workloads/scheduling-windows-workloads.adoc

```diff
@@ -0,0 +1,30 @@
+[id="scheduling-windows-workloads"]
+= Scheduling Windows container workloads
+include::modules/common-attributes.adoc[]
+:context: scheduling-windows-workloads
+
+toc::[]
+
+You can schedule Windows workloads to Windows compute nodes.
+
+[discrete]
+== Prerequisites
+
+* You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
+* You are using a Windows container as the OS image with the Docker container runtime add-on enabled.
```

Comment:
Docker-formatted container

Reply:
I have added `Docker-formatted container` language for this requirement and added the Docker deprecation/containerd note in each instance. This has been approved by PM.